tools/rum/explorer.html

```diff
@@ -105,6 +105,8 @@ <h2>TTFB</h2>
           <facet-sidebar>
             <list-facet facet="userAgent" drilldown="share.html">
               <legend>Device Type and Operating System</legend>
+              <a href="https://main--helix-website--adobe.aem.page/docs/rum-explorer#device-type-and-operating-system" class="help"
```

Comment:
We will not want to merge this as-is due to the branch.  I suggest we get the doc published first, then we can add the href links with the proper URL (probably as a path starting with `/docs` without a domain).

Reply:
Agree.